README.md

```diff
@@ -1 +1,140 @@
-# tor-graphql-client
\ No newline at end of file
+TOR GraphQL Client
+=======================
+
+This is a client library to communicate with the TOR application. You can use our Client to manage your assets in TOR.
+
+Usage of this API is limited to registered users only. If you would like to use TOR, please visit [travelbase.nl](https://travelbase.nl).
+
+
+---
+# Installation
+
+Run the following command to install the package using composer:
+
+```
+$ composer require oberonamsterdam/tor-graphql-client
+```
+
+# Usage
+
+To use this Client you need to define the API Key and endpoint as an .env variable or supply it when initiating the client class.
+```
+TOR_GRAPHQL_ENDPOINT="https://travelbase.nl/api/management/v2/graphql"
+TOR_GRAPHQL_APIKEY="MY_SECRET_APIKEY"
```

Comment:
Dit is in principe niet echt toepasselijk voor een library, gewoon meegeven aan de constructor, dat iemand die waardes uit zn environment of .env file haalt is eigen verantwoordelijkheid, wij moeten geen aannames gaan doen over hun applicatie setup.